### DISCUSSION TITLE
`Option + anything` -> `Control + anything` & `Option` alone -> Launchpad ?

### DISCUSSION THREAD

**Discussion by KaKi87** [pqrs-org/Karabiner-Elements#4068] [Q&A] [answered]
Hi,

I would like to :

- Map `Option` to `Control` when using it in a combination with something else (e.g. `Option + Command + S` becomes `Control + Command + S`) ;
- Map `Option` to *Launchpad* when pressed alone.

I know how to do either one, but how to do both at the same time ?

Simply adding both to *simple modifications* doesn't work.

Thanks

**Comment by tekezo** [comment, marked as answer]
You need to use Complex Modifications. This is a similar setting, so use it as a reference.
https://karabiner-elements.pqrs.org/docs/json/typical-complex-modifications-examples/#post-escape-if-left_control-is-pressed-alone

**Reply by KaKi87** [reply]
That works, thanks !
